homeassistant/components/verisure/binary_sensor.py

```diff
@@ -105,3 +106,42 @@ def available(self) -> bool:
     def device_class(self) -> str:
         """Return the class of this device, from component DEVICE_CLASSES."""
         return DEVICE_CLASS_CONNECTIVITY
+
+
+class VerisureFirmwareUpdate(CoordinatorEntity, BinarySensorEntity):
+    """Representation of a Verisure firmware."""
+
+    coordinator: VerisureDataUpdateCoordinator
+
+    @property
+    def name(self) -> str:
+        """Return the name of the binary sensor."""
+        return "Verisure Firmware update"
+
+    @property
+    def is_on(self) -> bool:
+        """Return the state of the sensor."""
+        return self.coordinator.is_upgradeable()
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self.coordinator.is_upgradeable() is not None
+
+    # pylint: disable=no-self-use
+    async def async_update(self) -> None:
+        """Update the state of the sensor."""
+        await self.coordinator.update_firmware_status()
```

Comment:
This call should be moved to the coordinator update method `_async_update_data`.

Reply:
This is not included because it doesn't refresh as often as `_async_update_data`, for now, it only refreshes once 1 day.

I think it doesn't make sense to update it every 2 minutes or so

Reply:
The method is throttled so it won't matter, I'm thinking.